compose/compose.go

```diff
@@ -647,11 +679,32 @@ func main() {
 		svc := getZero(i, fmt.Sprintf("idx=%d", i))
 		services[svc.name] = svc
 	}
-
+	custom_alphas := ""
+	if len(opts.CustomAlphaOptions) > 0 {
+		custom_alphas = opts.CustomAlphaOptions[0]
+	}
+	cust_index := 0
 	for i := 1; i <= opts.NumAlphas; i++ {
 		gid := int(math.Ceil(float64(i) / float64(opts.NumReplicas)))
 		rs := fmt.Sprintf("idx=%d; group=%d", i, gid)
-		svc := getAlpha(i, rs)
+
```

Comment:
Instead of parsing the custom flag in the for `loop`. Let's create a map upfront map[uint64]string -> Map from `alphaIndex` to `customFlag`. So we will not need any repetitive parsing in for loop and the code will look much simpler..

Reply:
okay